modules/renter/hostdb/hostdb.go

```diff
@@ -22,19 +22,22 @@ const (
 )
 
 var (
-	errNilCS     = errors.New("cannot create renter with nil consensus set")
-	errNilWallet = errors.New("cannot create renter with nil wallet")
-	errNilTpool  = errors.New("cannot create renter with nil transaction pool")
+	errNilCS     = errors.New("cannot create hostdb with nil consensus set")
+	errNilWallet = errors.New("cannot create hostdb with nil wallet")
+	errNilTpool  = errors.New("cannot create hostdb with nil transaction pool")
 )
 
 // The HostDB is a database of potential hosts. It assigns a weight to each
 // host based on their hosting parameters, and then can select hosts at random
 // for uploading files.
 type HostDB struct {
-	// modules
-	cs     modules.ConsensusSet
-	wallet modules.Wallet
-	tpool  modules.TransactionPool
+	// dependencies
+	wallet  wallet
+	tpool   transactionPool
+	dialer  dialer
+	sleeper sleeper
+	persist persister
+	log     logger
```

Comment:
It's not alphabetically ordered

Also, I'm not sure it makes sense to put the sleepr/dialer/persister/logger next to the wallet and tpool, I feel like it makes sense to keep the other modules separate.